api/commands/run.ts

```diff
@@ -133,7 +144,7 @@ export class Runner {
 
   private notifyListeners() {
     if (
-      Date.now() - flags.runnerNotificationPeriodMillis.get() <
+      Date.now() - this.flags.runnerNotificationPeriodMillis.get() <
```

Comment:
You don't want to reference either of these flags in this class anywhere anymore, use e.g `this.runnerNotificationPeriodMillis` instead (after adding the `private readonly` bit above)